backend/btrixcloud/crawlconfigs.py

```diff
@@ -1220,6 +1226,21 @@ async def run_now_internal(
             print(traceback.format_exc())
             raise HTTPException(status_code=500, detail=f"Error starting crawl: {exc}")
 
+    async def too_many_waiting_crawls_check(self, org: Organization):
```

Comment:
```suggestion
    async def check_if_too_many_waiting_crawls(self, org: Organization):
```
^ All of our other methods start with the verb, helps with readability